sci-libs/cantera/cantera-2.4.0.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD=90
+
+inherit desktop python-single-r1 scons-utils toolchain-funcs fortran-2
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cti cxx fortran pch +python static-libs test"
+
+REQUIRED_USE="
+	|| ( cxx python )
+	cti? ( ${PYTHON_REQUIRED_USE} )
+	fortran? ( cxx )
+	python? ( cti )
+	static-libs? ( cxx )
+	${PYTHON_REQUIRED_USE}
+	"
+
+RDEPEND="
+	python? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+	sci-libs/sundials:0=
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-cpp/eigen
+	dev-libs/boost
+	dev-libs/libfmt
+	python? (
+		dev-python/cython[${PYTHON_USEDEP}]
+	)
+	test? (
+		>=dev-cpp/gtest-1.8.0
+	)
+"
+
+PATCHES=( "${FILESDIR}/${PN}_${PV}_disable_debug_and_optimization.patch" )
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+	# modify SConstruct to comment block of lines and to set "env['libdirname'] = '$(get_libdir)'"
```

Comment:
Sorry, I have no idea how how the external patch-file must be look with dinamicaly changable variable `$(get_libdir)`. 

Also the patch for commenting part will be package version dependent.

Reply:
See, it's so unreadable I didn't even notice get_libdir in there! As for the libdir, the best option would be to make it possible to pass it as option.

Reply:
> I hope they aren't using it to build Python install path (because it differs per version on Gentoo, and can't be guessed). In any case, I presume that env['libdirname'] should indeed be forced somehow to match $(get_libdir). Note that there are other values possible, e.g. libx32 on x32 profile.

It seems that env['libdirname'] is use to install only the cantera C++ and Fortran libraries but not for python installation path that is auto-detected by the path of  python executable files. 

Anyway I made patch that allow to set this env variable by user.

Reply:
Done: replace sed patching to use `$(get_libdir)` with patch that allows to set `env['libdirname']` by user as SConstruct variable. This variable is set now in `src_install()` section.